libbpf-tools/cpufreq.c

```diff
@@ -155,6 +155,7 @@ static int init_freqs_hmz(__u32 *freqs_mhz, int nr_cpus)
 			fclose(f);
 			return -1;
 		}
+		freqs_mhz[i] /= 1000;
```

Comment:
Since you are here, please rename the function to `init_freqs_mhz` and add a comment to scaling_cur_freq saying that it is in kHZ.

Reply:
Thanks for helping review. The change is done.